Wikipedia/Code/ArticleContainerViewController.swift

```diff
@@ -18,26 +18,154 @@ class ArticleContainerViewController: ViewController {
         case loading
         case data
     }
-  
-    private var webViewController: ArticleWebViewController?
```

Comment:
I'd be fine with changing the name of this class to ArticleViewController, doesn't seem like the we need the distinction anymore.

Reply:
Sounds good - I'll make this change in mobile-html-ui-2 to avoid merge conflicts